# Command types
uint8 ON_OFF_TOGGLE = 0
uint8 IRIS_CLOSE = 1
uint8 IRIS_OPEN = 2
uint8 FOCUS_NEAR = 3
uint8 FOCUS_FAR = 4
uint8 ZOOM_WIDE = 5
uint8 ZOOM_TELE = 6
uint8 PAN_TILT = 9
uint8 STOP_ALL = 255


# Service variables
uint8 camera
uint8 cmd
int8 pan
int8 tilt
---
# Error types
uint8 BAD_COMMAND = 1
uint8 BAD_CAM_NUMBER = 2
uint8 CAMERA_INCAPABLE = 3
# Error = 0 when successful.
uint8 error
